src/api/merge.sh

```diff
@@ -6,6 +6,16 @@ build_dir="$3"
 output_lib="$4"
 shift 4
 
+if [ ! -e "$ar_binary" ]; then
+  echo "Error: ar_binary not found"
+  exit 1
+fi
+
+if [ ! -e "$libtool_binary" ]; then
+  echo "Error: libtool_binary not found"
+  exit 1
+fi
```

Comment:
Oops, sorry, spotted this too late and had to roll this back after merging: This will cause a regression since `libtool_binary` can be empty on systems where we don't use that tool.